_includes/splash.html

```diff
@@ -0,0 +1,19 @@
+<div id="carousel" class="carousel slide" data-ride="carousel">
+    <div class="carousel-inner">
+        <div class="carousel-item active">
+            <img src="{{ site.url}}/assets/images/marketing/1.jpg" />
+        </div>
+        {% for i in (2..12) %}
+        <div class="carousel-item">
+            <img src="{{ site.url}}/assets/images/marketing/{{ i }}.jpg" />
+        </div>
+        {% endfor %}
```

Comment:
Is this the best way to add images to the carousel? Can it not check for *all* images in the folder instead of 1 through 12? 

Not a big deal but if I add images I would add the image but also have to change `2..12` to `2..13`, right?

Reply:
I've pushed an update, all you have to do is to put images in the `assets/images/marketing` directory. Once it's done, Jekyll will search images automatically from the directory.